src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -132,6 +132,9 @@ message SmokeTestConfigProto {
 
   // A list describing the name and the value of the init fields.
   repeated string init_fields = 2;
+
+  // A list of the flattened parameters in the method to test.
+  repeated string flattened_parameters = 3;
```

Comment:
As discussed offline:

* In-language (e.g. Java), methods are matched up by parameter types, not parameter names.
* People may be surprised about seeing this either name-based or type-based matching specified in config.
* It will probably be easier for people to work with optionally giving names to flattening groups, and referencing that name here.